src/towncrier/test/test_packaging.py

```diff
@@ -1,22 +1,26 @@
 # Copyright (c) Amber Brown, 2015
 # See LICENSE for details.
 
-from incremental import Version
 from twisted.trial.unittest import TestCase
 
-from towncrier._version import _hatchling_version
+import towncrier
 
 
 class TestPackaging(TestCase):
-    def test_version_warning(self):
+    def test_version_attr(self):
```

Comment:
I am not sure why this change is needed.

The previous test look good enough to me. :)

I am misssing something ?

Reply:
The previous test was checking for the presence of an Incremental Version object (which obviously no longer exists in towncrier)

I've replaced it with a test to assert that a `DeprecationWarning` is present